GulajavaMinistudio/inferno
# Merge 3.5.0 (#1134)

* Even Dev and Master Branches (#1124)

* Bump version 3.4.3

* v3.4.3

* Inferno Mega Refactor (#1125)

* Mega refactor test

* Other files copied

* Update README.md

* Examples

* Update package.jsons

* Test different build process

* Fixed build

* Correct .travis.yml file

* Test against node 4,6,8

* Fix up karma

* Revert updates test

* Karma stuff

* Fix failing tests

* Change Travis and Coverage

* stuff

* Port 3.4.3

* Apply jestcodemods

* Jest fix

* Fix all tests

* fix merge

* Docs working

* Fix travis script

* Fix travis

* Fix travis script

* Remove custom travis script

* Fix package.json

* Set coverage minimums

* Remove inferno/test/utils

* Stupidly didn't test

* Remove lerna changelog

* prepublishOnly hook

* Testing github pages deploy

* Make fixture for packaging nicer

* Remove deploy

* Use npm@5

* Typedoc (#1126)

* Prepend module

* Prepend module files

* Lint precommit

* Add deploy targets on master branch

* Add npmignore into packages

* Activate prettier (#1130)

* Activate prettier

* Tidy

* Run prettier on travis

* Run prettier

* Remove write from prettier command

* prettier

* Remove prettier from travis

* Run prettier on all scripts

* Webpack fix

* Merge hydration fix

* Update and check scripts

* Update to 3.4.4

* Serialize prettier

* Fix odd versions

* Merge master (#1133)

* Inferno Redux @5  (#1131)

* Activate prettier

* Tidy

* Run prettier on travis

* Run prettier

* Remove write from prettier command

* prettier

* Remove prettier from travis

* Copy initial

* Tests nearly working

* Tests largely passing

* patch test utils

* patch test utils

* All tests passing

* Fix lint and tests

* remove old tests

* Fix karma tests

* imports

* Add docstring headers

* Merge master

* Fix update version

* Fix scripts

* v3.5.0

 *Before* submitting a PR please:
 - Include tests for the functionality you are adding! See CONTRIBUTING.md for details how to run tests.
 - Make sure you have based your branch off the [dev branch](https://github.com/infernojs/inferno/tree/dev).
 - Submit your PR against the [dev branch](https://github.com/infernojs/inferno/tree/dev).
 - Run `npm run build` and check that the build succeeds.
 - Ensure that the PR hasn't been submitted before.

---

## PR Template

**Objective**

This PR...

**Closes Issue**

It closes Issue #...
